### ISSUE TITLE
Aggregating local mask for an aggregate feature importance mask

### ISSUE THREAD

**Issue by ss8319** [dreamquark-ai/tabnet#581]
Hi guys, thank you for implementing the codebase for this.

I was just wondering if there is any feature to aggregate the local interpretability mask at every decision into a global decision mask.

I cant seem to find that function in the codebase/documentation

The original paper details combining the local mask 

<img width="1021" height="176" alt="Image" src="https://github.com/user-attachments/assets/09c7aea2-c2ef-4709-9453-346fcc556fdb" />

**Comment by Optimox** [comment]
The explanability method give you both the aggregated feature importance mask and the individual steps mask no ?
https://github.com/dreamquark-ai/tabnet/blob/2c0c4ebd2bb1cb639ea94ab4b11823bc49265588/pytorch_tabnet/tab_network.py#L190

**Comment by Optimox** [comment]
The global feature importance from the training set is accessible from using `clf.feature_importances_`
https://github.com/dreamquark-ai/tabnet/blob/2c0c4ebd2bb1cb639ea94ab4b11823bc49265588/pytorch_tabnet/abstract_model.py#L750

**Comment by ss8319** [comment]
Sure, thanks! is there a way to compute for the test set?

But it does look like simple edits to the code should be enough to make it
work too for test set.

**Comment by Optimox** [comment]
`clf._compute_feature_importances(X_test)` should work
